scripts/update_and_rebuild_libmesh.sh

```diff
@@ -38,6 +38,12 @@ if [[ -n "$METHOD" && -z "$METHODS" ]]; then
   export METHODS="$METHOD"
 fi
 
+# If the user has VTK available via our modules, we'll build with VTK
+VTK_OPTIONS=""
+if [[ ! -z "$VTKLIB_DIR" && ! -z "$VTKINCLUDE_DIR" ]]; then
```

Comment:
`! -z` is a double-negative.  Just use `-n` to test whether they are set.  See also, [this](http://stackoverflow.com/questions/3869072/test-for-non-zero-length-string-in-bash-n-var-or-var).